### PR TITLE
rmlui: Update assets after the update to master

### PR DESCRIPTION
**Pull request by illwieckz**
The branch is named `ishq/rmlui` despite not being mine because this way it's easier to find:

- https://github.com/Unvanquished/Unvanquished/pull/2027

I spotted that:

```
Warn: Deprecation notice: Element '<progressbar>' renamed to '<progress>', please adjust RML tags accordingly. 
Warn: Deprecation notice: Element '<progressbar>' renamed to '<progress>', please adjust RML tags accordingly. 
```

But that breaks HUD progress bars.

### CODE REVIEW

**Review by DolceTriade — APPROVED** [review]

**Comment by slipher** [comment]
This should target something other than master, shouldn't it?

**Comment by illwieckz** [comment]
Ah those are not silenced

```
Warn: Syntax error parsing inline property declaration 'opacity: -nan;'. 
Warn: Breaking change: The 'fill-image' property now needs to be set on the <progress> element, instead of its inner <fill> element. Please update your RCSS source to fix progress bars in this document. 
Warn: Breaking change: The 'fill-image' property now needs to be set on the <progress> element, instead of its inner <fill> element. Please update your RCSS source to fix progress bars in this document. 
Warn: Syntax error parsing inline property declaration 'opacity: -nan;'. 
```

It looks like I fixed the `fill-image` issue but not the `opacity: -nan`.

**Comment by illwieckz** [comment]
> This should target something other than master, shouldn't it?

Yes maybe, we can change the target anyway once we create a better one.

**Comment by illwieckz** [comment]
This may need this on Unvanquished side:

```
diff --git a/src/cgame/rocket/rocket.cpp b/src/cgame/rocket/rocket.cpp
index ace047a83..bff864a39 100644
--- a/src/cgame/rocket/rocket.cpp
+++ b/src/cgame/rocket/rocket.cpp
@@ -394,7 +394,7 @@ void Rocket_Init()
 
        // Add custom client elements
        RegisterElement<SelectableDataGrid>( "datagrid" );
-       RegisterElement<RocketProgressBar>( "progressbar" );
+       RegisterElement<RocketProgressBar>( "progress" );
        RegisterElement<RocketDataSelect>( "dataselect" );
        RegisterElement<RocketConsoleTextElement>( "console_text" );
        RegisterElement<RocketDataSourceSingle>( "datasource_single" );
```

But some progress bar are still full white, and they're always full.

Anyway this is low priority, I just wanted to try to know if that was easy or not.

File: ui/alien_build.rml [hunk 1]
```diff
@@ -125,7 +125,7 @@ tabs {
 </head>
 <body id="alien_build" class="circlemenu alien" onkeydown="CirclemenuHandleKey(event, document, AlienBuildableClick)" onshow="BuildAlienBuildMenu(document)">
 <tabset class="circlemenu">
-<tab></tab>
+<tab>Build</tab>
```
Comment (illwieckz) [inline ui/alien_build.rml:128]: You may want to do `<tab><translate>Build</translate></tab>` while you're at it, so we can translate the string later.

Same for `<tab>Evolve</tab>` below.

Comment (DolceTriade) [reply ui/alien_build.rml]: Done.

**Comment by illwieckz** [comment]
I can't approve because:

> Pull request authors can't approve their own pull requests.[^1]

But it LGTM.

[^1]: That's something GitLab allows anyway.

**Pull request merged**